alt_e2eshark/e2e_testing/test_configs/onnxconfig.py

```diff
@@ -231,3 +232,27 @@ def run(self, artifact: str, inputs: TestTensors, *, func_name=None) -> TestTens
                 raise FileNotFoundError(error_msg)
         return TestTensors.load_from(self.tensor_info_dict[test_name][0], self.tensor_info_dict[test_name][1], run_dir, "output")
 
+    def benchmark(self, artifact: str, inputs: TestTensors, repetitions: int = 5, *, func_name=None) -> float:
+        run_dir = Path(artifact).parent
+        detail_log = run_dir.joinpath("detail", "benchmark.detail.log")
+        commands_log = run_dir.joinpath("commands", "benchmark.commands.log")
+        report_json = run_dir.joinpath("detail", "benchmark.json")
```

Comment:
Can we edit cleanup flag level 3 to also include this benchmark.json file?

Reply:
Sure!